panel/src/components/Dialogs/LicenseDialog.vue

```diff
@@ -0,0 +1,67 @@
+<template>
+	<k-dialog
+		ref="dialog"
+		v-bind="$props"
+		@cancel="$emit('cancel')"
+		@submit="$emit('submit')"
+	>
+		<k-bar style="margin-bottom: var(--spacing-2)">
+			<h2 class="k-headline">
+				{{ $t("license") }}
+			</h2>
+			<k-button
+				icon="edit"
+				link="https://hub.getkirby.com"
+				target="_blank"
+				variant="filled"
+				size="xs"
+			>
+				Manage license
+			</k-button>
+		</k-bar>
+
+		<div class="k-table">
+			<table style="table-layout: auto">
+				<tbody>
+					<tr>
+						<th>Type</th>
+						<td>{{ license.type }}</td>
+					</tr>
+					<tr>
+						<th>Key</th>
+						<td class="k-text">
+							<code>{{ license.license }}</code>
+						</td>
+					</tr>
+					<tr>
+						<th>Purchased</th>
+						<td>{{ license.date }}</td>
+					</tr>
+					<tr>
+						<th>Activated</th>
```

Comment:
```suggestion
						<th>First Activated</th>
```

Reply:
Is "First" really needed here?

Reply:
Yes, because otherwise you could be confused that a new activation didn't update the date. For licensing, only the first activation is relevant.

Reply:
Maybe that gets to the root of it: Would people actually activate again when upgrading to additional years? Wouldn't the license code etc. all remain?

Reply:
At the moment, an upgrade generates a new license with a new license code. But that is not a necessity.

However in any case, you would need to activate your license again, otherwise the new Included Updates Period won't be in the license file and so Kirby would not be able to know that you are now allowed to use newer Kirby releases.

Reply:
Makes sense, otherwise we would need to communicate with the Hub constantly and we don't want that.

Reply:
I've completely removed it in the latest version. I think the license dialog should be as simple as possible and the hub can still have the info that we really want to show.